deploy/operator/libvirt_disks.sh

```diff
@@ -15,12 +15,13 @@ fi
 
 function create() {
     export SIZE=${SIZE:-50G}
+    export DISKS_IMAGE_DIR=${DISKS_IMAGE_DIR:-/tmp}
```

Comment:
maybe we can default to ``/var/tmp`` instead, so that we have bigger changes of getting enough space
@adriengentil wdyt? (or if you have other recommendations)

Reply:
Sounds good to me. Please just update the doc: https://github.com/openshift/assisted-service/blob/1e3df7d8791469f541e3268fb8d8f740b1897be7/deploy/operator/README.md#L55

Reply:
yes, `/var/tmp` would make more sense, but since this PR doesn't change the behavior, I would let it like that.